gui/tooltips.lua

```diff
@@ -0,0 +1,446 @@
+-- Show tooltips on units and/or mouse
+
+--@ module = true
+
+local RELOAD = false -- set to true when actively working on this script
+
+local gui = require('gui')
+local widgets = require('gui.widgets')
+local overlay = require('plugins.overlay')
+local ResizingPanel = require('gui.widgets.containers.resizing_panel')
```

Comment:
this doesn't need to be individually imported -- it can already be addressed as `widgets.ResizingPanel`